kolibri/core/assets/src/vue/icon-button.vue

```diff
@@ -1,6 +1,14 @@
 <template>
 
-  <button @click="$emit('click')" class="icon-button-scope" :class="{'primary' : primary, 'single-line': !showTextBelowIcon}">
+  <button 
+    type="button" 
+    @click="$emit('click')"
+    @keypress.enter.prevent = "$emit('enter')"
```

Comment:
Unexpected bug in Vue2 here. Prior to adding this line, if you hit `enter` while focused on the submit button for usercreate, the button would fire its `click` event.

Did some research, but most of the [results](http://stackoverflow.com/questions/12325066/button-click-event-fires-when-pressing-enter-key-in-different-input-no-forms) talked about how it had to do with the HTML spec (buttons are submit by default) and descriped how the solution could be curbed by adding a `type="button"` attribute.

This wasn't the case for me. Even after adding that line(4), click event was still firing. Started expirementing by creating buttons inside the modal and listening for click events on the whole document (Devtools are pretty sweet) and saw that it was indeed firing a `click` event on `enter` regardless of whether the attribute was present. This behavior was consistent across Chrome and Firefox.

Side note: same thing happens with the `<select>` element - after picking one with tab and hitting enter, a click event fires. Can test for yourself by typing `monitorEvents(document, "click");` into the console and seeing what elements cause it to fire.